src/components/cylc/Task.vue

```diff
@@ -37,7 +40,7 @@
           r="25"
           stroke-width="50"
           stroke-dasharray="157"
-          v-bind:style="progressStyle"
+          :style="`'stroke-dashoffset: ' + ${157 - (157 * (props.progress / 100))}`"
```

Comment:
I think this is how I originally implemented it but there was some problem with it not updating so it became a function.

Reply:
Hmmm, good to know. Let me test that.

Reply:
Thanks for the comment @oliver-sanders ! That saved us some regression-bug-fix. The single quote would cause a problem here. Pushed one more commit that fixes this.

![image](https://user-images.githubusercontent.com/304786/69666617-6a038600-10f1-11ea-945f-363e33b86d83.png)